### PR TITLE
some updates

### PR DESCRIPTION
**Pull request by yaananth**

### CODE REVIEW

**Comment by yaananth** [comment]
#1148 #1190

File: Tasks/AzurePowerShell/task.json [hunk 1]
```diff
@@ -64,15 +64,24 @@
...
+            "groupName":"advanced"
         }
     ],
     "instanceNameFormat": "Azure PowerShell script: $(ScriptPath)",
     "execution": {
-        "AzurePowerShell": {
-            "target": "$(currentDirectory)\\RunAzurePowerShell.ps1",
-            "argumentFormat": "",
-            "workingDirectory": "$(currentDirectory)"
+        "PowerShellExe": {
```
Comment (yaananth) [inline Tasks/AzurePowerShell/task.json]: oops this should be AzurePowerShell..updating now

File: Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1 [hunk 2]
```diff
@@ -233,7 +233,7 @@ if (!$azureService)
     $azureService = Invoke-Expression -Command $azureService
 }
 
-$diagnosticExtensions = Get-DiagnosticsExtensions $StorageAccount $CsCfg
+$diagnosticExtensions = Get-DiagnosticsExtensions $StorageAccount $serviceConfigFile
```
Comment (ericsciple) [inline Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1]: What if find-files doesn't resolve. I'm guessing we would error out, but just want to confirm that your change wouldn't introduce a worse error message.

File: Tasks/AzurePowerShell/RunAzurePowerShell.ps1 [hunk 3]
```diff
@@ -1,30 +0,0 @@
-[CmdletBinding(DefaultParameterSetName = 'None')]
-param
-(
-    [String] [Parameter(Mandatory = $true)]
```
Comment (ericsciple) [inline Tasks/AzurePowerShell/RunAzurePowerShell.ps1]: Are we losing arg validation by ditching this intermediate script?

File: Tasks/AzurePowerShell/task.json [hunk 4]
```diff
@@ -64,15 +71,24 @@
...
         }
     ],
     "instanceNameFormat": "Azure PowerShell script: $(ScriptPath)",
     "execution": {
         "AzurePowerShell": {
-            "target": "$(currentDirectory)\\RunAzurePowerShell.ps1",
-            "argumentFormat": "",
-            "workingDirectory": "$(currentDirectory)"
+            "target": "$(ScriptPath)",
+            "argumentFormat": "$(ScriptArguments)",
```
Comment (ericsciple) [inline Tasks/AzurePowerShell/task.json]: Is this how other AzurePowerShell tasks do it? Does this change introduce any differences in how the args are processed?

Comment (yaananth) [inline Tasks/AzurePowerShell/task.json]: good point, I am reverting this change

File: Tasks/AzurePowerShell/RunAzurePowerShell.ps1 (hunk 3 above)
Comment (yaananth) [inline Tasks/AzurePowerShell/RunAzurePowerShell.ps1]: per offline discussion, I am reverting this change

File: Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1 (hunk 2 above)
Comment (yaananth) [inline Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1]: I will test and update

**Pull request closed without merging**